src/relayselector.py

```diff
@@ -270,12 +273,12 @@ def main():
     args = parse_cmd_args()
 
     exits = get_exits(args.data_dir,
-                      country_code = args.countrycode,
```

Comment:
I think I prefer the previous version. This was also discussed here: https://github.com/NullHypothesis/exitmap/pull/48#issuecomment-206876969